src/frontend/src/hooks/users.hooks.ts

```diff
@@ -89,3 +90,21 @@ export const useUpdateUserSettings = () => {
     }
   );
 };
+
+export const useUpdateUserRole = () => {
+  const auth = useAuth();
```

Comment:
instead of doing this whole "making sure they're logged in" thing, we should modify the endpoint to not take in the submitter Id and instead use the auth getCurrentUser function that checks the logged in user

Reply:
thanks for the tip I was thinking there had to be a better way to do that.